src/index.html

```diff
@@ -15,8 +15,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="logo.png"
+          alt="moyo_logo"
+          class="img"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
```

Comment:
Don't use spaces in `<a>` tag's `href` property. If these are placeholder links, consider using `#apple`, `#samsung`, etc., to indicate the section they would navigate to.